src/braintree/apple/braintreeOnPaymentAuthorizedApple.ts

```diff
@@ -87,7 +87,8 @@ export async function braintreeOnPaymentAuthorizedApple(event: ApplePayPaymentAu
             token: nonce,
             gateway_public_id: gatewayPublicId,
             currency: currencyCode,
-            amount: totalAmountDue
+            amount: totalAmountDue,
+            wallet_pay_type: 'applepay',
```

Comment:
Was create payment endpoint updated to reflext this change?

If so, we need to update the documentation https://developer.boldcommerce.com/api/checkout#tag/Payments/operation/CreatePayment

Reply:
https://github.com/bold-commerce/developer-docs/pull/95/files